docs/reference/docs/get.asciidoc

```diff
@@ -42,7 +42,7 @@ The API also allows to check for the existence of a document using
 
 [source,js]
 --------------------------------------------------
-HEAD twitter/_doc/0
```

Comment:
The document CRUD APIs are retaining the `_doc` path component (so this actually shouldn't change).